### PR TITLE
Add style guide for perl

### PR DESCRIPTION
**Pull request by aishwarya-sankhla**
Fixes issue: #2303 

Changes:
Adds the style guide for perl.

### CODE REVIEW

**Comment by arnavb** [comment]
This is in the wrong folder. Put it in a new folder called `perl`.

**Comment by aishwarya-sankhla** [comment]
I have moved it to the perl folder.

**Comment by arnavb** [comment]
You have a duplicate file in the wrong directory.

**Comment by AdiChat** [comment]
Kindly place the code snippets in code tags in proper format. It will be good to merge then 👍

File: guides/coding_style/perl/style.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+Perl Code Style Guidelines
+
+Indentation:
+Tabs should be used to indent all code blocks.  Spaces should never be used to indent code blocks.  Mixing tabs and spaces results in misaligned code blocks for other developers who prefer different indentation settings.
+
```
Comment (arnavb) [inline guides/coding_style/perl/style.md]: Add \```perl at the beginning of the snippet, and \``` to the end.

**Comment by arnavb** [comment]
#2033

**Comment by arnavb** [comment]
@AdiChat You said this would be good to merge once the code tags were added, so....

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**